src/navigation/navigationContainer.tsx

```diff
@@ -98,6 +104,8 @@ const TabNavigator = () => {
 };
 
 export const NavigationWrapper = () => {
+  const config = useConfig();
+
   const currentCamera = useAppDataStore(state => state.currentCamera);
```

Comment:
The Header Button should be defined before the actual nav component.
```suggestion
const HeaderLeftIcon = ({tintColor}: HeaderBackButtonProps) => {
  const nav = useNavigation();
  const config = useConfig();

  if (!config.data?.birdseye.enabled) {
    return null;
  }

  return (
    <BirdseyeIcon
      height={25}
      width={25}
      onPress={() => {
        nav.navigate('Birdseye');
      }}
      fill={tintColor}
      fillSecondary={tintColor}
    />
  );
};
```